### ISSUE TITLE
Not detecting Memu Play

### ISSUE THREAD

**Issue by toraku72** [Back-X/anti-vm#3]
The APK does not detect Memu Play as VM
[screenshot](https://imgur.com/mInaIkv)

**Comment by graysuit** [comment]
@toraku72 Which apk you used ? 
did you compile it ? or took from somewhere here ?

and also which MeMu version you use ?
I tried on 6.0.x.x, and it detected.

![2](https://user-images.githubusercontent.com/54432616/139595001-ffc55daf-b657-47f9-8fdd-3cafea1dc35d.PNG)
![1](https://user-images.githubusercontent.com/54432616/139595003-555cd107-57e2-4dbf-8ef5-3d80aadc7f7e.PNG)


Try latest apk and let me know: https://github.com/Back-X/anti-vm/actions/runs/1182942330
Watch for `andro_release`, download, extract and use that apk.
Thanks

**Comment by toraku72** [comment]
@graysuit The APK I used was the anti-vm.apk from https://github.com/Back-X/anti-vm/releases/download/1/anti-vm.apk and I'm using Memu latest version 7.5.6
Maybe Memu has updated their app to generate random values for the getRadioVersion check? It's the only emulator I tested that failed to detect.

**Comment by swagkarna** [comment]
@toraku72 refer this https://gist.github.com/ShinJJang/9fac5e779f23d478a234

**Comment by swagkarna** [comment]
![vm](https://user-images.githubusercontent.com/46685308/139792351-d70a5c0f-fced-45d8-97b6-34a41af391a4.png)
@graysuit

**Comment by swagkarna** [comment]
getRadioVersion();
1. Nox
![Screenshot (99)](https://user-images.githubusercontent.com/46685308/139799272-af00f21e-ef73-4b7f-b1bb-4d1e107b6440.png)

BaseBandversion==null // vm detected

2. Memu (latest version)
![Screenshot (207)](https://user-images.githubusercontent.com/46685308/139799501-61d5bf3a-8e36-40a8-bada-19b23c8fdb46.png)
![Screenshot (208)](https://user-images.githubusercontent.com/46685308/139799515-f44c5985-f253-47aa-a1d6-6ad4c00163de.png)

BaseBandversion != null // vm check fail

3. cloud device
![Screenshot (200)](https://user-images.githubusercontent.com/46685308/139799636-9b11a56f-8fc3-4211-b532-42b29e12ec8f.png)

 BaseBandversion != null // vm check fail
 
 ```
 
Baseband is firmware that controls all radio devices on the phone, including GPS, wi-fi, Bluetooth, NFC, cell data and voice, etc. because these functions are highly regulated and certified, they are separate from the main operating system. AP software version is characterized by the Android version and kernel version.
```

**Comment by graysuit** [comment]
>  Maybe Memu has updated their app to generate random values for the getRadioVersion check? 

You are right. In new Memu versions, getRadioVersion() do have some value. So anti_vm script will fail here.

**Comment by graysuit** [comment]
> @toraku72 refer this https://gist.github.com/ShinJJang/9fac5e779f23d478a234

This is static method and works for some vms only.

**Comment by graysuit** [comment]
> 
> 
> getRadioVersion();
> 
>     1. Nox
>        ![Screenshot (99)](https://user-images.githubusercontent.com/46685308/139799272-af00f21e-ef73-4b7f-b1bb-4d1e107b6440.png)
> 
> 
> BaseBandversion==null // vm detected
> 
>     1. Memu (latest version)
>        ![Screenshot (207)](https://user-images.githubusercontent.com/46685308/139799501-61d5bf3a-8e36-40a8-bada-19b23c8fdb46.png)
>        ![Screenshot (208)](https://user-images.githubusercontent.com/46685308/139799515-f44c5985-f253-47aa-a1d6-6ad4c00163de.png)
> 
> 
> BaseBandversion != null // vm check fail
> 
>     1. cloud device
>        ![Screenshot (200)](https://user-images.githubusercontent.com/46685308/139799636-9b11a56f-8fc3-4211-b532-42b29e12ec8f.png)
> 
> 
> BaseBandversion != null // vm check fail
> 
> ```
> 
> Baseband is firmware that controls all radio devices on the phone, including GPS, wi-fi, Bluetooth, NFC, cell data and voice, etc. because these functions are highly regulated and certified, they are separate from the main operating system. AP software version is characterized by the Android version and kernel version.
> ```

Baseband looks promising. But require heavy testing on many emulators. So that we can verify its stability.
